build_tools/packaging/linux/package.json

```diff
@@ -207,103 +256,2443 @@
+    "Description_Short": "HIP:Heterogenous-computing Interface for Portability",
+    "Description_Long": "HIP:Heterogenous-computing Interface for Portability [RUNTIME - AMD]",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-systems",
+    "Components": [
+      "lib",
+      "run",
+      "doc"
+    ],
+    "Includes": [
+      "hip-runtime-amd"
+    ],
+
+    "Artifact": "core-hip",
+    "Gfxarch": "False",
+    "Composite": "Yes"
+
+  },
+
+  {
+    "Package": "rocm-hip-devel",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "libc6",
+      "file",
+      "rocm-hip-runtime",
+      "amd-llvm"
+    ],
+    "RPMRequires": [
+      "glibc",
+      "file",
+      "rocm-hip-runtime",
+      "amd-llvm"
+    ],
+    "Maintainer": "HIP Support <hip.support@amd.com>",
+    "Description_Short": "Heterogenous-computing Interface for Portability",
+    "Description_Long": "Heterogenous-computing Interface for Portability [DEVELOPMENT]",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-systems",
+    "Components": [
+      "dev"
+    ],
+    "Includes": [
+      "hip-devel"
+    ],
+    "Artifact": "core-hip",
+    "Gfxarch": "False",
+    "Composite": "Yes"
+
+  },
+
+  {
+    "Package": "rocm-blas",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": "AMD library for BLAS, sparse and solver in ROCm",
+    "Description_Long": "blas, sparse and solver libraries",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-libraries",
+    "Components": [
+      "lib",
+      "run",
+      "doc"
+    ],
+    "Includes": [
+      "rocsparse",
+      "hipsparse",
+      "hipsolver",
+      "rocsolver",
+      "hipblas",
+      "rocblas",
+      "hipblaslt"
+    ],
+    "Artifact": "blas",
+    "Gfxarch": "True",
+    "Composite": "Yes"
+  },
+
+  {
+    "Package": "rocm-blas-devel",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6",
+      "rocm-blas"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc",
+      "rocm-blas"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": "AMD library for BLAS, sparse and solver in ROCm",
+    "Description_Long": "blas, sparse and solver development files",
```

Comment:
Why is BLAS capitalized in the short description but not the long description?

Reply:
Capitalized BLAS